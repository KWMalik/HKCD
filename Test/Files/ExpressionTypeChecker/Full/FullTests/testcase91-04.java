//invalid assignment hidden in a branch
class boring
{
   public static void main( String[] args )
   {
      System.out.println(0);
   }
}

class UV {
	int a;
	public int method(int test){
		return 0;
	}
}

class shadow extends UV{
	int b;
	public int method(int test){
		int result=0;
		if (3 + 2) {
			boolean b=false;
			result=b;
		} else {
			result=1;
		}
	}
}