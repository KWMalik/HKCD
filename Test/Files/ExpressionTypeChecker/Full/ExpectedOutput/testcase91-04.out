

ERROR in line 27 and column 2: Syntax error



ERROR in line 28 and column 2: Couldn't repair and continue parse

Exception processing file: Can't recover from previous error(s)
